js_tests/admin/SelectFilter2.test.js

```diff
@@ -16,3 +16,131 @@ QUnit.test('init', function(assert) {
     assert.equal($('.selector-remove').text(), "Remove");
     assert.equal($('.selector-clearall').text(), "Remove all");
 });
+
+QUnit.test('filtering available options', function(assert) {
```

Comment:
`'filtering available options'` works without the patch and should be committed separately.

Reply:
Good spot. I had them originally as a separate commit, but then got too eager with the flattening of the commits. 

Updated. 👍🏻